config/rootfs/debos/overlays/chromeos-flash/usr/bin/chromeos-flash-modules.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+set -xe
+
+mount_special_fs()
+{
+    local base="${1:-/root/chromeos}"
+
+    mount udev -t devtmpfs "$base"/dev
+    mount proc -t proc "$base"/proc
+    mount sysfs -t sysfs "$base"/sys
+    mount -t tmpfs tmpfs "$base"/tmp
+    mount
+}
+
+umount_special_fs()
+{
+    local base="${1:-/root/chromeos}"
+
+    umount "$base"/dev
+    umount "$base"/proc
+    umount "$base"/sys
+    umount "$base"/tmp
+    mount
+}
+
+flash_chromeos()
+{
+    local dev
+    local mount_dir=$(basename "$IMAGE_MOUNTPOINT")
+    local root_path=$(dirname "$IMAGE_MOUNTPOINT")
+
+    echo "Setting up Chrome OS chroot..."
+    mkdir -p "$IMAGE_MOUNTPOINT"
+    cd "$root_path"
+    dev=$(losetup -P -f --show "$IMAGE_NAME")
+    mount "$dev"p3 "$mount_dir" -o ro
+    mount_special_fs "$IMAGE_MOUNTPOINT"
+    mount --bind . "$mount_dir"/mnt/empty
+    ls -l "$mount_dir"/mnt/empty/"$IMAGE_NAME"
+    ls -l "$mount_dir"/dev/mmc*
+
+    echo "Starting to flash..."
+    chroot "$mount_dir" \
+         /usr/sbin/chromeos-install \
+        --dst /dev/mmcblk0 \
+        --payload_image /mnt/empty/"$IMAGE_NAME" \
+        --yes
+    sync
+
+    umount_special_fs "$IMAGE_MOUNTPOINT"
+    umount "$mount_dir"/mnt/empty
+    umount "$mount_dir"
+    losetup -d "$dev"
+}
+
+enable_rw()
+{
+    local mount_dir=$(basename "$IMAGE_MOUNTPOINT")
+    local root_path=$(dirname "$IMAGE_MOUNTPOINT")
+
+    echo "Making the rootfs writeable..."
+    mkdir -p "$IMAGE_MOUNTPOINT"
+    cd "$root_path"
+    mount /dev/mmcblk0p3 "$mount_dir" -o ro
+    mount_special_fs "$IMAGE_MOUNTPOINT"
+    cd "$mount_dir"
+    chroot . \
+        /usr/share/vboot/bin/make_dev_ssd.sh \
+        --remove_rootfs_verification \
+        --force
+    cd "$root_path"
+    umount_special_fs "$IMAGE_MOUNTPOINT"
+    umount "$mount_dir"
+    sync
+    partprobe /dev/mmcblk0
+}
+
+fixup_tmpfiles()
```

Comment:
BTW this _nearly_ works, and it's a workaround. We should leave it in the script for now, but I'm just mentioning this here as the resulting rootfs image won't be bootable as-is. Still we can check flashing works and merge this PR, then fix the stateful partition issue as a follow-up.